src/releng_common/releng_common/cors.py

```diff
@@ -7,9 +7,10 @@
 from flask_cors import CORS
 
 
-cors = CORS()
-
-
 def init_app(app):
+    # Allow specific origins
+    origins = app.config.get('CORS_ORIGINS', '*')
```

Comment:
i think by default we should just allow everything but rather turn this around. be very restrictive eg. no cors if none CORS_ORIGINS is set.